ding/framework/supervisor.py

```diff
@@ -167,7 +167,7 @@ def __init__(
         super().__init__(proc_id, init, *args, shared_object=shared_object, **kwargs)
         self._thread = None
 
-    def start(self, recv_queue: queue.Queue):
+    def start(self, recv_queue: queue.Queue, _):
```

Comment:
maybe change `_` to `**kwargs`?

Reply:
Done, and this PR rolls back all changes to "ding/framework/supervisor.py". "supervisor.py" now should be clean.